src/content/en/updates/2018/10/background-fetch.md

```diff
@@ -0,0 +1,448 @@
+        </ul>
+      </td>
+    </tr>
+    <tr>
+      <td><code>failureReason</code></td>
+      <td><p>One of the following:</p>
+        <ul>
+          <li><code>""</code> - The background fetch hasn't failed.</li>
+          <li><code>"aborted"</code> – The background fetch was aborted by the user, or
+          <code>abort()</code> was called.</li>
+          <li><code>"bad-status"</code> - One of the responses had a not-ok status, e.g. 404.</li>
+          <li><code>"fetch-error"</code> - One of the fetches failed for some other reason, e.g.
+          CORS, MIX, an invalid partial response, or a general network failure for a fetch that
+          cannot be retried.</li>
+          <li><code>"quota-exceeded"</code> - Storage quota was reached during the background
+          fetch.</li>
+          <li><code>"download-total-exceeded"</code> - The provided `downloadTotal` was
+          exceeded.</li>
+        </ul>
+      </td>
+    </tr>
+    <tr>
+      <td><code>recordsAvailable</code></td>
+      <td><code>boolean</code><br>Can the underlying requests/responses can be accessed.</td>
+    </tr>
+    <tr><th colspan=2>Methods</th></tr>
+    <tr>
+      <td><code>abort()</code></td>
+      <td>Returns <code>Promise&lt;boolean&gt;</code><br>Abort the background fetch.
+      <p>The returned promise resolves with true if the fetch was successfully aborted.</p></td>
+    </tr>
+    <tr>
+      <td><code>matchAll(request, opts)</code></td>
+      <td>Returns <code>Promise&lt;Array&lt;BackgroundFetchRecord&gt;&gt;</code><br>Get the requests
+      and responses.
+      <p>The arguments here are the same as
+      <a href="https://developer.mozilla.org/en-US/docs/Web/API/Cache/match#Parameters">the cache
+      API</a>. Calling without arguments returns a promise for all records.</p>
+      <p>See below for more details.</p></td>
+    </tr>
+    <tr>
+      <td><code>match(request, opts)</code></td>
+      <td>Returns <code>Promise&lt;BackgroundFetchRecord&gt;</code><br>As above, but resolves with
+      the first match.</td>
+    </tr>
+    <tr><th colspan=2>Events</th></tr>
+    <tr>
+      <td><code>progress</code></td>
+      <td>Fired when any of <code>uploaded</code>, <code>downloaded</code>, <code>result</code>, or
+      <code>failurereason</code> change.</td>
+    </tr>
+  </tbody>
+</table>
+
+### Tracking progress
+
+This can be done via the `progress` event. Remember that `downloadTotal` is whatever value you
+provided, or `0` if you didn't provide a value.
+
+    bgFetch.addEventListener('progress', () => {
+      // If we didn't provide a total, we can't provide a %.
+      if (!bgFetch.downloadTotal) return;
+
+      const percent = Math.round(bgFetch.downloaded / bgFetch.downloadTotal * 100);
+      console.log(`Download progress: ${percent}%`);
+    });
+
+### Getting the requests and responses
+
+Caution: In Chrome's current implementation you can only get the requests and responses during
```

Comment:
This might also be a good spot to say that anything requiring CORS preflight is currently blocked by Chrome's implementation, and will fail.